Hestia/Hestia/frontend/resources/Globals.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using Foundation;
+using UIKit;
+using System.Drawing;
+
+using System.Collections;
+using CoreGraphics;
+using Hestia.backend;
+using Hestia.backend.models;
+
+namespace Hestia.DevicesScreen.resources
+{
+    public static class Globals
+    {
+        public static bool LocalLogin { get; set; }
+        public static ServerInteractor LocalServerinteractor { get; set; }
+        public static String ServerName { get; set; }
+        public static int Port { get; set; }
+        public static String IP { get; set; }
+
```

Comment:
1 white line, 2 white lines, none, this file could use a little makeover, or explanation why you would want it like that.

Reply:
Looked not really nice indeed,  hope b80ff119b44ea8c46ec0e9d7ec8ed40b82801901 is better

Reply:
Indeed better, thanks.